code/index.html

```diff
@@ -4,19 +4,169 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+
+           <div class="navigationbar"> <nav>
+          <a href="#home" class="active"><picture>
+            <img src="./logo.svg" alt="technigo">
+           </picture></a>
+          <div id="myLinks">
+           <a href="#">SIGTHSEEING</a>
```

Comment:
It might be just a preference thing, but instead of writing the navigations links in capital letter I like to style them in CSS using text-transform: uppercase;